scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/Consumer.java

```diff
@@ -0,0 +1,70 @@
+package com.scylladb.cdc.model.worker;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.function.BiConsumer;
+
+/**
+ * Opaque wrapper around any of the available consumer interfaces
+ * 
+ * @author calle
+ *
+ */
+public class Consumer {
```

Comment:
I know that you update `Printer` application in your next PR to use this new API (and new "batch sequence" API), but you should also update root-level `README.md` and `scylla-cdc-printer/README.md` to document the new API (currently the old one is documented).

Reply:
Done.